alchemist-full/build.gradle.kts

```diff
@@ -30,6 +37,99 @@ application {
     mainClass.set("it.unibo.alchemist.Alchemist")
 }
 
+// JPackage
+tasks.jpackage {
+    // General info
+    resourceDir = "${rootProject.projectDir}/package-settings"
+    appName = rootProject.name
+    appVersion = rootProject.version.toString().substringBefore('-')
+    copyright = "Copyright (C) 2010-2023, Danilo Pianini and contributors"
+    appDescription = rootProject.description
+    vendor = ""
+    licenseFile = "${rootProject.projectDir}/LICENSE.md"
+    verbose = isInCI
+
+    // Packaging settings
+    input = rootProject.layout.buildDirectory.map { it.dir("package-input") }.get().asFile.path
+    destination = rootProject.layout.buildDirectory.map { it.dir("package") }.get().asFile.path
+    mainJar = "alchemist-full-${rootProject.version}-all.jar"
+    mainClass = "it.unibo.alchemist.Alchemist"
+
+    linux {
+        icon = "${rootProject.projectDir}/package-settings/logo.png"
+        type = ImageType.RPM
```

Comment:
Which alternatives to we have? RPM, MSI, PKG, and what else?

Reply:
Linux: 
 - rpm
 - deb

Windows:
- msi
- exe

MacOs:
- pkg
- dmg

Reply:
Is there any reason not to generate all of them?

Reply:
I do not think the plugin supports multiple types. In order to achieve it we should run the task, reconfigure it by changing the type property then re-run it

Reply:
Is there a task type that we can configure? If so, we can generate our custom tasks.